apps/fz_http/lib/fz_http/config.ex

```diff
@@ -67,4 +75,7 @@ defmodule FzHttp.Config do
       end
     end
   end
+
+  # String.to_atom/1 is only used in test env
+  defp key_function(app, key), do: String.to_atom("#{app}-#{key}")
```

Comment:
@jamilbk just move `defp key_function` inside `else` block above and static analysis won't complain about it.

Reply:
D'oh, that's where I thought it was defined initially. Thanks for the second pair of eyes.